_health-care/_js/components/financial-assessment/Child.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+
+import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
+import ErrorableSelect from '../form-elements/ErrorableSelect';
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import DateInput from '../form-elements/DateInput';
+import FullName from '../questions/FullName';
+import SocialSecurityNumber from '../questions/SocialSecurityNumber';
+
+import { childRelationships } from '../../utils/options-for-select.js';
+import { isBlank, isValidMonetaryValue } from '../../utils/validations';
+
+// TODO: create unique nodes for each child in applicationData
+
+class Child extends React.Component {
+  render() {
+    const message = 'Please enter only numbers and a decimal point if necessary (no commas or currency signs)';
+
+    return (
+      <div>
+        <div className="row">
+          <div className="small-12 columns">
+            <p>Child's Name</p>
```

Comment:
This was a hack so we could tell what's going on.  It's totally the wrong UI.

I imagine the right UI should be to have some sort of roll-up in the table with a "remove" button on the side in place of this index and then only render the lower component on a click or something (either in a modal or a panel, or an accordian or ...whatever you people who understand UX think should happen :P)

For now, will just change it to a remove button.